.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,12 @@
 <!--  Thanks for sending a pull request! 
-If this is your first time, please read our contributor guidelines: https://github.com/Dynatrace/dynatrace-configuration-as-code/blob/main/CONTRIBUTING.md
+If this is your first time, please read our contributor guidelines: https://github.com/Dynatrace/dynatrace-configuration-as-code/blob/main/CONTRIBUTING.md  -->
```

Comment:
Maybe we can get rid of that. I assume that users interested in contributing would probably find it on their own.

Reply:
Happily!